pkg/kudoctl/cmd/install_test.go

```diff
@@ -23,3 +25,22 @@ func TestNewCmdInstallReturnsCmd(t *testing.T) {
 		t.Fatal("child and root commands should have the same normalization functions")
 	}
 }
+
+func TestNewCmdInstall_WithParameters(t *testing.T) {
+	newCmdInstall := NewInstallCmd()
+
+	newCmdInstall.Flags().Set("parameter", "foo")
+	err := newCmdInstall.RunE(newCmdInstall, []string{})
+	assert.NotNil(t, err, "a parameter without value worked")
+
+	newCmdInstall = NewInstallCmd()
+	newCmdInstall.Flags().Set("parameter", "bar=")
+	err = newCmdInstall.RunE(newCmdInstall, []string{})
+	assert.NotNil(t, err, "a parameter with empty value worked")
+
+	newCmdInstall = NewInstallCmd()
+	newCmdInstall.Flags().Set("parameter", "foo=bar")
```

Comment:
We should also check the happy path — passing just a valid parameter should not return an error =).

Reply:
Yes, but maybe that could even be another table test for https://github.com/kudobuilder/kudo/pull/283

Reply:
> We should also check the happy path — passing just a valid parameter should not return an error =)

Been there - a happy path would actually try to install something and will fail someplace else ;) The fact that parameter validation is not separated from command execution in cobra is lamentable.

Reply:
Areed. I'm ok with this being resolved. We have plans to separate out a better package for this anyway (as @zen-dog and I talked about in another thread in this PR)